recipes/libglvnd/meta.yaml

```
@@ -0,0 +1,152 @@
+{% set version = "1.7.0" %}
+
+package:
+  name: libglvnd-split
+  version: {{ version }}
+
+source:
+  url: https://gitlab.freedesktop.org/glvnd/libglvnd/-/archive/v{{ version }}/libglvnd-v{{ version }}.tar.bz2
+  sha256: d0e1925a3c9aee0143b8c181ac31d5637c8faa081759c277b8e16c7075612c11
+  patches:
+    - libglvnd_look_for_egl_icd_in_system_location.patch
+
+build:
+  skip: true  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ stdlib('c') }}
+    - meson
+    - pkg-config
+    - ninja
+  host:
+    - xorg-libx11
+    - xorg-libxext
+    - xorg-xextproto
+    - xorg-glproto
+
+outputs:
+  - name: libglvnd
+    script: build_libglvnd.sh
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ stdlib('c') }}
+        - meson
+        - pkg-config
+        - ninja
+      host:
+        - xorg-libx11
+        - xorg-libxext
+        - xorg-xextproto
+        - xorg-glproto
+    test:
+      commands:
+        - test -f $PREFIX/lib/libEGL.so.1
+        - test -f $PREFIX/lib/libGL.so.1
+        - test -f $PREFIX/lib/libGLESv1_CM.so.1
+        - test -f $PREFIX/lib/libGLESv2.so.2
+        - test -f $PREFIX/lib/libGLX.so.0
+        - test -f $PREFIX/lib/libOpenGL.so.0
+        - test -f $PREFIX/lib/libGLdispatch.so.0
+        - test ! -f $PREFIX/lib/pkgconfig/egl.pc
+        - test ! -f $PREFIX/lib/pkgconfig/gl.pc
+        - test ! -f $PREFIX/lib/pkgconfig/glesv1_cm.pc
+        - test ! -f $PREFIX/lib/pkgconfig/glesv2.pc
+        - test ! -f $PREFIX/lib/pkgconfig/glx.pc
+        - test ! -f $PREFIX/lib/pkgconfig/opengl.pc
+        - test ! -f $PREFIX/lib/pkgconfig/libglvnd.pc
+        - test ! -f $PREFIX/include/EGL/egl.h
+        - test ! -f $PREFIX/include/EGL/eglext.h
+        - test ! -f $PREFIX/include/EGL/eglplatform.h
+        - test ! -f $PREFIX/include/GL/gl.h
+        - test ! -f $PREFIX/include/GL/glcorearb.h
+        - test ! -f $PREFIX/include/GL/glext.h
+        - test ! -f $PREFIX/include/GL/glx.h
+        - test ! -f $PREFIX/include/GL/glxext.h
+        - test ! -f $PREFIX/include/GLES/egl.h
+        - test ! -f $PREFIX/include/GLES/gl.h
+        - test ! -f $PREFIX/include/GLES/glext.h
+        - test ! -f $PREFIX/include/GLES/glplatform.h
+        - test ! -f $PREFIX/include/GLES2/gl2.h
+        - test ! -f $PREFIX/include/GLES2/gl2ext.h
+        - test ! -f $PREFIX/include/GLES2/gl2platform.h
+        - test ! -f $PREFIX/include/GLES3/gl3.h
+        - test ! -f $PREFIX/include/GLES3/gl31.h
+        - test ! -f $PREFIX/include/GLES3/gl32.h
+        - test ! -f $PREFIX/include/GLES3/gl3ext.h
+        - test ! -f $PREFIX/include/GLES3/gl3platform.h
+        - test ! -f $PREFIX/include/KHR/khrplatform.h
+        - test ! -f $PREFIX/include/glvnd/GLdispatchABI.h
+        - test ! -f $PREFIX/include/glvnd/libeglabi.h
+        - test ! -f $PREFIX/include/glvnd/libglxabi.h
```

I'm not against an explicit list per se, but I think the duplication between `libglvnd` (negative tests) and `libgl-devel` (positive tests) is detrimental. It's a matter of taste (non-binding suggestion), but I'd prefer something like:

```suggestion
        # absence of headers (need to go into -devel output)
        {% set headers = [
            'EGL/egl.h', 'EGL/eglext.h', 'EGL/eglplatform.h',
            'GL/gl.h', 'GL/glcorearb.h', 'GL/glext.h', 'GL/glx.h', 'GL/glxext.h',
            'GLES/egl.h', 'GLES/gl.h', 'GLES/glext.h', 'GLES/glplatform.h',
            'GLES2/gl2.h', 'GLES2/gl2ext.h', 'GLES2/gl2platform.h',
            'GLES3/gl3.h', 'GLES3/gl31.h', 'GLES3/gl32.h', 'GLES3/gl3ext.h',
            'GLES3/gl3platform.h', 'KHR/khrplatform.h',
            'glvnd/GLdispatchABI.h', 'glvnd/libeglabi.h', 'glvnd/libglxabi.h'
        ] %}
        {% for each_header in headers %}
        - test ! -f $PREFIX/include/{{ each_header }}
        {% end for %}
```